src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/ParticipantDto.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests.Invitations
+{
+    public class ParticipantDtoGet
```

Comment:
splitt ParticipantDtoGet  and ParticipantDtoEdit into 2 sep files. Same name on file as class